App/localization/languages/fr.json

```
@@ -1,5 +1,87 @@
 {
-  "home_header_change_location": "Changer de lieu",
-  "error_screen_common_sorry": "Désolé!\n",
-  "error_screen_error_cannot_load_cigarettes": "Nous n'avons pas pu\ncharger vos\ncigarettes."
+    "error_screen_common_sorry": "Désolé !",
+    "error_screen_error_cannot_load_cigarettes": "Ne ne pouvons\npas charger\nvos cigarettes.",
+    "error_screen_choose_other_location": "Choisir un autre lieu",
+    "error_screen_error_description": "Il y a soit un problème avec nos bases de données, ou bien vous n'avez pas de station de surveillance de l'air près de vous. Réessayez plus tard !",
+    "error_screen_error_message": "Erreur: {{errorText}}",
+    "home_station_too_far_message": "Nous ne sommes pas parvenus à trouver une station plus proche de vous. Les résultats pourraient être inexacts à cette distance.",
+    "home_share_title": "Saviez-vous que vous fumez peut-être jusqu'à 20 cigarettes par jour, juste parce-que vous vivez dans une grande ville ?",
+    "home_share_message": "Zut ! J'ai 'fumé' {{cigarettes}} cigarettes aujourd'hui en respirant l'air urbain. Et toi ? Regarde ici : https://shootismoke.github.io",
+    "home_header_air_quality_station_distance": "Station de surveillance de l'air: {{distanceToStation}}km",
+    "home_btn_why_is_station_so_far": "Pourquoi la station est-elle si éloignée?",
+    "home_btn_see_detailed_info": "Voir des informations plus détaillées",
+    "home_btn_more_details": "Plus de détails",
+    "home_btn_faq_about": "Faq/À propos",
+    "home_btn_share": "Partager",
+    "home_common_you_ll_smoke": "Vous allez fumer",
+    "home_common_you_smoked": "Vous avez fumé",
+    "home_common_oh": "Oh",
+    "home_common_cigarette": "Cigarette",
+    "home_common_cigarettes": "Cigarettes",
+    "home_smoked_cigarette_title": "{{swearWord}}! {{presentPast}} <{{cigarettes}} {{singularPlural}}> aujourd'hui.",
+    "home_swear_word_dang": "Oh wow !",
```

```suggestion
    "home_swear_word_dang": "Oh wow",
```

since we already add a `!` after